jmix-core/core/src/main/java/io/jmix/core/impl/FileTransferServiceImpl.java

```diff
@@ -51,6 +51,8 @@ public class FileTransferServiceImpl implements FileTransferService {
 
     @Autowired
     private FileStorageLocator fileStorageLocator;
+    @Autowired
+    private CoreProperties coreProperties;
```

Comment:
No more usage - remove field.
No actual changes in this file (import refactoring is not a part of the issue and should not present here).